### ISSUE TITLE
Geoserver

### ISSUE THREAD

**Issue by Isiaka-Ibroheem** [ernest19/Geospatial_Developers#3]
Hi, I have been having a recurring problem whenever I enter the localhost:8080/geoserver on my browser, it keeps bringing "problem loading page".
I followed the video on installing geoserver but i have not been able to host or work on it because of the error message.

**Comment by ernest19** [comment]
please start the menu and search for  "Start Geeoserver"  click on it to start the geoserver then you can try accessing it again in your browser. will be on standby

**Comment by Isiaka-Ibroheem** [comment]
I just did, still, it didn't work.

**Comment by Isiaka-Ibroheem** [comment]
I even uninstall my previous java to install the exact one you installed in the tutorial video.

**Comment by ernest19** [comment]
> I even uninstall my previous java to install the exact one you installed in the tutorial video.

kindly install java version 8  and point the geoserver to that directory

**Comment by Isiaka-Ibroheem** [comment]
Please can you guide me on how i will point to the directory. That seems to be the problem now.

**Comment by Isiaka-Ibroheem** [comment]
Thank you, Sir, I have been able to fix it. 
I really appreciate it.

**Comment by Isiaka-Ibroheem** [comment]
![Screenshot 2022-08-19 120728](https://user-images.githubusercontent.com/110978360/185605909-465bddc9-7a08-4eb6-8fed-c06b48dc2e38.jpg)

**Comment by Isiaka-Ibroheem** [comment]
I kept getting this error when i tried adding a store to my geoserver.

**Comment by ernest19** [comment]
Please the your postgis database .the issue is coming from that angle.better still let have a call to look at it together.

**Comment by Diyende** [comment]
@ernest19 I have grouped my layers including the geotiff but previewing it has been a challenge. The feedback i got says " the request used more time than allowed and has forcefully stopped. Max rendering time is 60s"

**Comment by ernest19** [comment]
> @ernest19 I have grouped my layers including the geotiff but previewing it has been a challenge. The feedback i got says " the request used more time than allowed and has forcefully stopped. Max rendering time is 60s"

Please  please are trying to say you have publish the layers but  you are unable to review it

**Comment by Diyende** [comment]
yes, exactly my point. I'm publishing my grouped layers which includes the geotiff but unable to publish it
